c2rust-transpile/src/translator/builtins.rs

```diff
@@ -691,16 +684,9 @@ impl<'c> Translation<'c> {
         let mem = mk().path_expr(vec!["libc", name]);
         let args = self.convert_exprs(ctx.used(), args)?;
         args.and_then(|args| {
-            let mut args = args.into_iter();
-            let dst = args
-                .next()
-                .ok_or("Missing dst argument to convert_libc_fns")?;
-            let c = args
-                .next()
-                .ok_or("Missing c argument to convert_libc_fns")?;
-            let len = args
-                .next()
-                .ok_or("Missing len argument to convert_libc_fns")?;
+            let [dst, c, len]: [_; 3] = args
+                .try_into()
+                .expect("`convert_libc_fns` must have exactly 3 arguments");
```

Comment:
```suggestion
                .expect("`convert_libc_fns` must have exactly 3 arguments: [dst, c, len]");
```

Unlike the other two, the error messages here specify the argument names, so I'd keep them.